src/Compilers/CSharp/Portable/SourceGeneration/CSharpGeneratorDriver.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("ApiDesign", "RS0016:Add public types and members to the declared API", Justification = "In Progress")]
```

Comment:
How are we tracking the public API changes? Do we add a prototype per suppression or an issue to follow up on all of them?

Reply:
I'll make an issue to track all of them. The API is expected to vary wildly over the first few PRs and I didn't want to have to keep updating the API files.

---
In reply to: [381640798](https://github.com/dotnet/roslyn/pull/41789#discussion_r381640798) [](ancestors = 381640798)